### DISCUSSION TITLE
Set up QField Cloud on own server

### DISCUSSION THREAD

**Discussion by pggsn** [opengisch/QField#4293] [Q&A]
Hi everyone,

I am new to QField (Cloud) and am charged with implementing a local QField Cloud on our own company server.
Is there already any guide how to do this or can someone give advice from their own experience?

I figured that the [README](https://github.com/opengisch/qfieldcloud/blob/master/README.md) gives an overview of how to install QField Cloud - do i "just" need to clone the repository to our server?

Any advices are highly appreciated! Looking forward to using QField and QField Cloud in our daily mapping business :-)

Best
Kim

**Comment by pol-todo** [comment]
Hi @pggsn ,

I too am looking for instructions. Was curious if you found them?

Best,
Paul

**Comment by pol-todo** [comment]
@pggsn Found it, it is located here: https://github.com/opengisch/qfieldcloud

**Reply by pggsn** [reply]
Hi @paulterinho,
yes, I found the instructions. However, I asked our company IT to do the installation and as far as I know, some problems occured (unfortunately, I don't know what kind of). Up to know, they are solved and our local instance is all set :-) Did you manage to set it up?
Best,
Kim

**Comment by pol-todo** [comment]
Hi Kim,

I'll be tackling that today, great to know it worked locally! 

Did they get it hooked up to QField app on mobile yet? I haven't yet looked at this piece of the puzzle. Was curious about how complicated it was to set up. 

Good morning,
Paul

**Comment by pol-todo** [comment]
(oh sorry too! I just reread your original post and saw that you already found that README, sorry about that! 🙈)

**Comment by mygiano** [comment]
any detail instruction? i had installed and run properly to sync project to the cloud, but nothing shown after synchronize from Qfield app. Maybe someone here can help, where the missing step. Thanks

**Comment by suricactus** [comment]
As pointed earlier in the conversation, everything you need to know is in the README file in https://github.com/opengisch/qfieldcloud . In case you want dedicated support to install QFieldCloud on your own infrastructure, please send us an email on sales [at] qfield.cloud .
